px-data-grid-paginated.html

```diff
@@ -128,31 +188,16 @@
             },
 
             /**
-             * Number of items fetched at a time from the dataprovider.
+             * The number of items fetched during each request to the
+             * `remoteDataProvider`. See the `remoteDataProvider` property
+             * for more information.
```

Comment:
I assume this property also controls the number of records displayed in a page, regardless of whether you use remote data or local data.